meteor/server/api/rest/rest.ts

```diff
@@ -88,14 +87,9 @@ Meteor.startup(() => {
 				docString += `/:${paramName}`
 			})
 
-			assignRoute('GET', resource, docString, (args) => {
+			assignRoute('GET', resource, docString, async (args) => {
```

Comment:
The lambda function is made async, but if the promise fails, then this will result in an unhandled promise rejection. Possibly rejected promises should be caught here.

Reply:
I have added a catch to handle the awaited promise.

Reply:
I don't think that is necessary, as the lambda is already awaited inside of a catch block. https://github.com/tv2/tv-automation-server-core/blob/SOF-1321/fixSofieAPIforHealthMonitor/meteor/server/api/rest/rest.ts#L159-L168

By adding that catch here, if the promise does reject then the request will return `200 OK` with an empty array, rather than reporting the error to the caller

Reply:
I'll remove it then :-) Thanks!